### PR TITLE
Gibs takedown

### PR DESCRIPTION
**Pull request by cvalarida**
Takes down the GI Bill Chapter 33 Benefits app and replaces it with this error message.

image

**Note:** The user doesn't have to log in before seeing this.

### CODE REVIEW

**Review by markgreenburg — APPROVED** [review]
👍

File: content/pages/education/gi-bill/post-9-11/ch-33-benefit-backup.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Check Benefit
+entryname: post-911-gib-status
+layout: page-react.html
+includeBreadcrumbs: true
+in_maintenance: false
```
Comment (markgreenburg) [inline content/pages/education/gi-bill/post-9-11/ch-33-benefit-backup.md]: We might just be able to toggle this guy instead of changing the content

Comment (cvalarida) [reply content/pages/education/gi-bill/post-9-11/ch-33-benefit-backup.md]: :man_facepalming:

**Comment by cvalarida** [comment]
Well that was dumb. I'm gonna just make another branch to avoid reverting anything here.

:disappointed:

**Pull request closed without merging**